worker/handler.ts

```diff
@@ -78,6 +80,111 @@ const ga: Reporter = async (...args) => {
   await Promise.all([gaForBots(...args), gaForHumans(...args)]);
 };
 
+function ga4(
+  request: Request,
+  conn: ConnInfo,
+  response: Response,
+  _start: number,
+  error?: unknown,
+) {
+  Promise.resolve().then(async () => {
+    // We're tracking page views and file downloads. These are the only two
+    // HTTP methods that _might_ be used.
+    if (!/^(GET|POST)$/.test(request.method)) {
+      return;
+    }
+
+    const bot = isBot(request);
+    const botName = bot ? getBotName(request) : null;
+
+    // If the visitor is using a web browser, only create events when we serve
+    // a top level documents or download; skip assets like css, images, fonts.
+    if (!bot && !isNonAssetRequest(request, response) && error == null) {
+      return;
+    }
+
+    // Filter out traffic generated by uptime checks (Checkly and GCP).
+    if (
+      botName === "Checkly" ||
+      botName === "GoogleStackdriverMonitoring-UptimeChecks"
+    ) {
+      return;
+    }
```

Comment:
Maybe these should be filtered out elsewhere (i.e. during analysis)

If we rely on this, then a new check is added, or the UA changes or checkly, then the data will be thrown off and we won't really understand why.